sdk/deviceprovisioningservices/azure-resourcemanager-deviceprovisioningservices/src/main/test/com/azure/resourcemanager/deviceprovisioningservices/AllocationPolicyTests.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.deviceprovisioningservices;
+
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.resourcemanager.deviceprovisioningservices.fluent.models.ProvisioningServiceDescriptionInner;
+import com.azure.resourcemanager.deviceprovisioningservices.models.AllocationPolicy;
+import com.azure.resourcemanager.deviceprovisioningservices.models.IotDpsPropertiesDescription;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.resources.models.ResourceGroup;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AllocationPolicyTests extends DeviceProvisioningTestBase
+{
+    @Test
+    @DoNotRecord(skipInPlayback = true)
```

Comment:
Why not record?

Reply:
This is the standard in track 2 java these days. You can see that our ADT test files look the same: https://github.com/Azure/azure-sdk-for-java/blob/996d2de5991f16382a545dba2e4227161f0b7e9d/sdk/digitaltwins/azure-resourcemanager-digitaltwins/src/test/java/com/azure/resourcemanager/digitaltwins/DigitalTwinsLifecycleTests.java#L34

Reply:
As for why, @weidongxu-microsoft can comment on that since he wrote these ADT tests